server/neptune/workflows/activities/github.go

```diff
@@ -229,7 +229,7 @@ func (a *githubActivities) FetchRoot(ctx context.Context, request FetchRootReque
 	if err != nil {
 		return FetchRootResponse{}, errors.Wrap(err, "fetching and extracting zip")
 	}
-	localRoot := terraform.BuildLocalRoot(request.Root, request.Repo, destinationPath)
+	localRoot := terraform.BuildLocalRoot(request.Root, request.Repo, filepath.Join(destinationPath, request.Root.Path))
```

Comment:
I think this means we won't clean up the entire clone https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/neptune/workflows/activities/cleanup.go#L23

Reply:
Good catch! Fixed this